configuration/authentication.mdx

```diff
@@ -202,9 +202,26 @@ Once the user has authenticated with Google, they will be redirected to the addr
 Google's consent screen can be configured to only accept accounts that are within your Google Workspace organization.
 
 Other providers have similar mechanisms for attenuating who can leverage this authentication flow.
-
 </Tip>
 
+### Method: Kubernetes
+
+The `kubernetes` method provides the ability to exchange Kubernetes service account tokens for client tokens.
+
+```yaml config.yaml
+authentication:
+  methods:
+    kubernetes:
+      enabled: true
+      discovery_url: https://kubernetes.default.svc.cluster.local
+      ca_path: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+      service_account_token_path: /var/run/secrets/kubernetes.io/serviceaccount/token
+```
+
+Once enabled, client tokens can be retrieved by sending a Kubernetes pod's service account token to the [VerifyServiceAccount](/reference/authentication/kubernetes-verify-service-account) operation in the API.
```

Comment:
do we need to add this file to the reference docs?

Reply:
Ah yes, yes we do. I can do the openapi defs before we merge.

Reply:
The PR is up here https://github.com/flipt-io/flipt-api/pull/45

Reply:
I think this is the other missing piece too https://github.com/flipt-io/flipt-docs/pull/38
Once that is merged I think this is good to go.